PluginsAndFeatures/azure-toolkit-for-intellij/src/main/java/com/microsoft/azure/toolkit/intellij/connector/database/DatabaseResourcePanel.java

```diff
@@ -95,10 +82,20 @@ private void init() {
         testConnectionActionPanel.setVisible(false);
         testResultTextPane.setEditable(false);
         testConnectionButton.setEnabled(false);
-        envPrefixTextField.setText(definition instanceof SqlServerDatabaseResource.Definition ? "AZURE_SQL_" : "AZURE_MYSQL_");
         this.passwordSaveComboBox.setValue(Arrays.stream(Password.SaveType.values())
                 .filter(e -> StringUtils.equals(e.name(), Azure.az().config().getDatabasePasswordSaveType())).findAny()
                 .orElse(Password.SaveType.UNTIL_RESTART));
+        // server loader
+        this.serverComboBox.setItemsLoader(() -> Objects.isNull(this.serverComboBox.getSubscription()) ? Collections.emptyList() :
+                Azure.az(AzureMySql.class).subscription(this.serverComboBox.getSubscription().getId()).list());
```

Comment:
why this common database panel is bounded to MySQL?

Reply:
updated.